StripePaymentSheet/StripePaymentSheet/Resources/JSON/form_specs.json

```diff
@@ -768,23 +768,7 @@
             "dark_theme_png": "https://js.stripe.com/v3/fingerprinted/img/payment-methods/icon-pm-amazonpay_dark-9e42e4498e0bee515c423ef66010852f.png",
             "dark_theme_svg": "https://js.stripe.com/v3/fingerprinted/img/payment-methods/icon-pm-amazonpay_dark-d3ef2eeefed43cea1f969c3a2a4118d8.svg"
             },
-        "fields": [
-        ],
-        "next_action_spec": {
```

Comment:
The next action for Amazon Pay is now handled by STPPaymentHandler to support polling.